### PR TITLE
Load Reporting filter v0

### PR DESCRIPTION
**Pull request by dgquintas**
no-op load reporting filter. To be plugged in to census.

### CODE REVIEW

**Comment by dgquintas** [comment]
@jboeuf I made some changes to https://github.com/grpc/grpc/pull/7637/files#diff-688faa01dd269343d626342a0646b170 Not sure if that's something you'd be happy with, but seemed the right place to do it (and allows testing more paths).

File: src/core/ext/load_reporting/load_reporting_filter.c [hunk 1]
```diff
@@ -156,37 +158,36 @@ static void init_channel_elem(grpc_exec_ctx *exec_ctx,
                               grpc_channel_element *elem,
                               grpc_channel_element_args *args) {
   GPR_ASSERT(!args->is_last);
-
   channel_data *chand = elem->channel_data;
   memset(chand, 0, sizeof(channel_data));
 
-  chand->id = (intptr_t)args->channel_stack;
+  const grpc_auth_context *auth_context =
```
Comment (jboeuf) [inline src/core/ext/load_reporting/load_reporting_filter.c]: Assuming that this filter is server-side (which i'm pretty sure it is), you should grab the auth context from the `grpc_call_element_args` in `init_call_elem` like this:
```
grpc_server_security_context* server_context = args->context[GRPC_CONTEXT_SECURITY].value;
grpc_auth_context* auth_context = NULL;
if (server_context != NULL) {
  auth_context = server_context->auth_context;
}
  
```

Comment (dgquintas) [inline src/core/ext/load_reporting/load_reporting_filter.c]: Thanks! What's the advantage/difference wrt the current implementation? I see your suggestion does it per-call. Can the auth context or the peer identity change within the same channel?

**Comment by ctiller** [comment]
LGTM

**Comment by dgquintas** [comment]
test this please

**Comment by ctiller** [comment]
is this still a thing?

**Comment by dgquintas** [comment]
Yes. I'm going to merge and go make an offering on the altar of Jenkins to see if this time things pass...

**Comment by dgquintas** [comment]
test this please

**Comment by dgquintas** [comment]
test this please

**Comment by dgquintas** [comment]
Closing. This has gone way too stale. It'll be revisited in the future.

**Comment by thelinuxfoundation** [comment]
Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.

After the following items are addressed, please respond with a new comment here, and the automated system will re-verify.

* One or more commits are not associated with a GitHub user. The email used in the Git commit must match a verified email address in a GitHub user profile. For more information, please see https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/

_Regards,
CLA GitHub bot_

**Pull request closed without merging**